README.md

```diff
@@ -11,49 +11,373 @@
 
-```swift
-import FaunaDB
-..
-.
-```
+Apart of the FaunaDB driver, we also provide [RxSwift](https://github.com/ReactiveX/RxSwift) reactive programming [extensions](RxSources/Client+Rx.swift) for FaunaDB.
+
+> working on another language? Take a look at our github account, we provide Fauna DB drivers for many popular languages and many others are coming. Don't hesitate to contact us if the language you are working on is not supported yet. Notice you can always use the [REST API](https://faunadb.com/documentation/rest) directly.
+
+## About Fauna BD
+
+FaunaDB is a state of the art db system that aims to be as reliable, secure and fast as possible. For more information about Fauna, please visit our [website](https://faunadb.com/). If you are curious about Fauna DB design and architecture, please check out the [Design and Architecture of FaunaDB](https://faunadb.com/pdf/Design%20and%20Architecture%20of%20FaunaDB%2020160701.pdf) whitepaper.
 
 ## Requirements
 
 * iOS 8.0+
 * Xcode 7.3.1+
+* Swift 2.2
 
-## Getting involved
+## Getting started
 
-* If you **want to contribute** please feel free to **submit pull requests**.
-* If you **have a feature request** please **open an issue**.
-* If you **found a bug** or **need help** please **check older issues and [FAQ](#faq) before submitting an issue.**.
+1. Install the driver first. We can install the driver either by using [Carthage](https://github.com/Carthage/Carthage), [CocoaPods](https://cocoapods.org/) or manually as Embedded Framework. Go to [Installation](#installation) section to know how.
 
-Before contribute check the [CONTRIBUTING](https://github.com/faunadb/faunadb-swift/blob/master/CONTRIBUTING.md) file for more info.
+2. If you haven’t already, [sign up for an account](https://faunadb.com/#signup-modal) to start using FaunaDB.
 
-If you use **FaunaDB** in your app We would love to hear about it! Drop us a line on [twitter](https://twitter.com/faunadb).
+3. From here, Fauna DB [tutorial](https://faunadb.com/tutorials) will walk you through step by step how to get started with Fauna DB.
 
-## Examples
+#### Basic Usage
 
-Follow these 3 steps to run Example project: Clone FaunaDB repository, open FaunaDB workspace and run the *Example* project.
+```swift
+import Foundation
+import Result
+import FaunaDB
 
-You can also experiment and learn with the *FaunaDB Playground* which is contained in *FaunaDB.workspace*.
+// create a fauna client
+// Logger will shows in the Xcode console the curl representation of each message sent to FaunaDB, it also shows the fauna response data. This is useful during app development.
 
-## Installation
+let secret = <YOUR_CLIENT_SECRET>
+let dbName = <YOUR_DB_NAME>
+var client = Client(secret: secret, observers: [Logger()])
 
-#### CocoaPods
+func setUpSchema(callback: (Result<Value, Error> -> ()){
```

Comment:
@erickpintor  Are you talking about examples that js and c# driver illustrate in the readme?